src/main/java/horseshoe/TemplateLoader.java

```diff
@@ -326,21 +327,51 @@ private Template loadPartial(final String name, final Stack<Loader> loaders) thr
 	 * @throws ReflectiveOperationException if an error occurs while dynamically creating and loading an expression
 	 */
 	private TemplateRenderer loadPartial(final Stack<Loader> loaders, final TemplateLoadState state, final String tag) throws LoadException, ReflectiveOperationException {
-		final Matcher matcher;
-		String indentation = null;
+		final boolean doubleArrow = tag.length() > 1 && tag.charAt(1) == '>'; // ">>" uses indentation on first line, no trailing new line
+		final String indentation;
 
-		if (tag.length() > 1 && tag.charAt(1) == '>') { // ">>" uses indentation on first line, no trailing new line
-			matcher = LOAD_PARTIAL_PATTERN.matcher(tag).region(2, tag.length());
+		if (doubleArrow) {
 			state.setStandaloneStatus(TemplateLoadState.TAG_CHECK_TAIL_STANDALONE);
+			indentation = null;
 		} else {
-			matcher = LOAD_PARTIAL_PATTERN.matcher(tag).region(1, tag.length());
 			indentation = state.removeStandaloneTagHead();
 
 			// 1) Standalone tag uses indentation for each line, no trailing new line
 			// 2) Non-standalone tag uses indentation on first line, with trailing new line
 			state.setStandaloneStatus(indentation == null ? TemplateLoadState.TAG_CANT_BE_STANDALONE : TemplateLoadState.TAG_CHECK_TAIL_STANDALONE);
 		}
 
+		final Matcher expressionMatch = PARTIAL_EXPRESSION.matcher(tag);
+		if (expressionMatch.matches()) {
+			final String op = expressionMatch.group(1);
+			final String expr = expressionMatch.group(2);
+			return new TemplateRenderer(result -> {
+				try {
+					return loadPartial(loaders, state, op + String.valueOf(result), indentation, doubleArrow);
```

Comment:
Should we have this load the partial or just do a look-up of local partials? I don't really like loading a new partial from a file at render-time.

Reply:
I like the consistency of behavior from a user's perspective.